packages/validator/src/commands/convertCmd.ts

```diff
@@ -9,27 +10,52 @@ import { invalidConversionMessage } from '../utils/messages';
 const oas2ois = 'OAS2OIS.json';
 const ois2config = 'OIS2Config.json';
 
-if (process.env.npm_config_template) {
-  console.log(
-    JSON.stringify(convert(process.env.npm_config_specs || process.argv[3], process.env.npm_config_template), null, 2)
-  );
+const args = yargs(hideBin(process.argv))
+  .option('from', {
+    description: 'Name of the source airnode specification format',
+    demandOption: 'to',
+    default: '',
+    string: true,
+  })
+  .option('to', {
+    description: 'Name of the target airnode specification format',
+    demandOption: 'from',
+    default: '',
+    string: true,
+  })
```

Comment:
I think you can use [`choices`](https://github.com/yargs/yargs/blob/master/docs/api.md#choiceskey-choices) for these two options, can't you?

Reply:
No, because in that case `''` must be one of the options and yargs would display this option in hints even though `''` is not a valid conversion format.